Userland/DevTools/HackStudio/LanguageServers/ClientConnection.h

```diff
@@ -35,6 +35,9 @@ class ClientConnection
     virtual void auto_complete_suggestions(GUI::AutocompleteProvider::ProjectLocation const&) override;
     virtual void find_declaration(GUI::AutocompleteProvider::ProjectLocation const&) override;
 
+    static void set_declarations_of_document_callback(ClientConnection&, const String&, Vector<GUI::AutocompleteProvider::Declaration>&&);
+    static void set_todo_entries_of_document_callback(ClientConnection&, const String&, Vector<String>&&);
+
```

Comment:
Are these functions needed here in `ClientConnection.h`? I don't think they're ever used :thinking:

Reply:
Solved :)